spec/system/applying/passported_on_age_spec.rb

```diff
@@ -0,0 +1,105 @@
+require 'rails_helper'
+
+RSpec.describe 'Apply for Criminal Leagal Aid when under 18 years old' do
+  describe 'Submitting a completed application' do
+    before do
+      visit root_path
+      click_button('Start now')
+
+      # steps/provider/select_office
+      choose('2A555X')
+      # prevent requests to the datastore for counters for tab headings on the next page
+      allow_any_instance_of(Datastore::ApplicationCounters).to receive_messages(returned_count: 0)
+      save_and_continue
+
+      # applications
+      click_link('Start an application')
+      choose('New application')
+      save_and_continue
+
+      # edit (Task list)
+      click_link('Client details')
+
+      # steps/client/details
+      fill_in('First name', with: 'Jo')
+      fill_in('Last name', with: 'Bloggs')
+      fill_date('Date of birth', with: 17.years.ago.to_date)
+      save_and_continue
+
+      # steps/client/is_application_means_tested
+      save_and_continue
+
+      # steps/client/case_type
+      choose('Indictable')
+      save_and_continue
+
+      # steps/client/residence_type
+      choose('They do not have a fixed home address')
+      save_and_continue
+
+      # steps/client/contact_details
+      choose('Provider’s office')
+      save_and_continue
+
+      # steps/client/urn
+      save_and_continue
+
+      # steps/case/has_the_case_concluded
+      choose_answer('Has the case concluded?', 'No')
+      save_and_continue
```

Comment:
Would be good to add a shared context or example to reuse some of the parts of the form that would be repeated in a lot of specs (e.g. case details)

Reply:
Potentially, I'd rather extract those patterns when we get to that point.